docs/ruby_api.md

```diff
@@ -25,6 +25,8 @@ The full API is documented below.
 * `Prism.load(source, serialized)` - load the serialized syntax tree using the source as a reference into a syntax tree
 * `Prism.parse_comments(source)` - parse the comments corresponding to the given source string and return them
 * `Prism.parse_file_comments(source)` - parse the comments corresponding to the given source file and return them
+* `Prism.parse_success?(source)` - parse the syntax tree corresponding to the given source string and return true if it was parsed without errors or warnings
```

Comment:
It's quite unusual to consider warnings a failure.
Also https://rubydoc.info/gems/prism/Prism%2FParseResult:success%3F only mentions errors.

I think `Prism.parse_success?(source)` should be equivalent to `Prism.parse(source).success?` and the check should be no errors.
This method should not enable warnings by default, but I guess that's the default for `verbose` anyway?